packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md

```diff
@@ -1,60 +1,89 @@
 ﻿# Angular for XMCloud
 
-> Sitecore JSS Angular App for XM Cloud. For the current release this feature is experimental.
+> Sitecore JSS Angular App for XM Cloud. For the current release this feature is in public beta.
```

Comment:
"For the current release this feature is in public beta" > "This feature is currently in public beta."